src/eth.rs

```diff
@@ -0,0 +1,24 @@
+/// Ethernet header, which is present at the beginning of every Ethernet frame.
+#[repr(C, packed)]
+#[derive(Debug, Copy, Clone)]
+pub struct EthHdr {
+    /// Destination MAC address.
+    pub dst_hwaddr: [u8; 6],
```

Comment:
why not just dst_addr and src_addr?

Reply:
To differentiate between MAC addresses and IP addresses. I liked Dave's suggestion about that. https://github.com/vadorovsky/network-types/pull/3#discussion_r1056447135